client/lib/products-values/get-jetpack-product-tagline.js

```diff
@@ -8,13 +8,16 @@ import { getJetpackProductsTaglines } from 'lib/products-values/translations';
 /**
  * Get Jetpack product tagline based on the product purchase object.
  *
- * @param   product {object}             Product purchase object
- * @returns         {string|object} Product tagline
+ * @param   product  {object}           Product purchase object
+ * @param   isOwned  {boolean}          Whether the site owns the product
+ * @returns          {string|object} 	Product tagline
  */
-export function getJetpackProductTagline( product ) {
+export function getJetpackProductTagline( product, isOwned = false ) {
 	product = formatProduct( product );
 	assertValidProduct( product );
 	const jetpackProductsTaglines = getJetpackProductsTaglines();
 
-	return jetpackProductsTaglines?.[ product.product_slug ];
+	const productTagline = jetpackProductsTaglines?.[ product.product_slug ];
+
+	return isOwned ? productTagline?.owned || productTagline?.default : productTagline?.default;
```

Comment:
Not all translations object has the `owned` key, s, we need to check if it exists. If not, we return the `default` key content.